src/test/java/TestRunner.java

```diff
@@ -0,0 +1,13 @@
+import org.junit.runner.*;
```

Comment:
This whole file is failing, maybe just condense it and add it to the file StringManipulationTest.java

Reply:
When I got the other stuff to work and ran your TestRunner.java all it printed was "true".

Reply:
Probably don't need this file then.

Reply:
I'll remove the file. Thanks for reviewing.